store/src/db/tests.rs

```diff
@@ -382,6 +416,9 @@ fn test_db_account() {
 fn test_notes() {
     let mut conn = create_db();
 
+    let block_num1 = 1;
```

Comment:
Would the idiomatic way to write this var name be `block_num_1` ?

Reply:
I prefer it the idiomatic way personally.

Reply:
changed